dbms/src/DataStreams/MergingSortedBlockInputStream.h

```diff
@@ -77,7 +79,7 @@ class MergingSortedBlockInputStream : public IProfilingBlockInputStream
 protected:
     struct RowRef
     {
-        ColumnRawPtrs columns;
+        ColumnRawPtrs * columns = nullptr;
```

Comment:
It looks redundant a little bit.
Can we use `ColumnRawPtrs & columns() { return shared_block->all_columns; }` instead?
Amount of the pointer dereferences should be the same.